std_msgs/Header header
float64 fluid_pressure              # Absolute pressure reading in Pascals.
float64 fluid_pressure_variance     # 0 is interpreted as variance unknown
